Add News entry "Mixxx 2.3.6 released" (#303)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md

title: Mixxx 2.3.6 released
authors: Daniel Schürmann
tags: 2.3, 2.3.6, release announcement
status: draft
comments: yes

Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.

> This little bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and ensuring a reliable mixing experience.

Reviewer: ```suggestion
This small bug fix release may appear insignificant, but it manifests our effort to deliver rock-solid quality and ensure a reliable mixing experience.
```
'little' sounds a bit like 'cute'

Reviewer 2: I agree with the `s/little/small` and the `s/ensuring/to ensure/` but I prefer the "testament" version of the rest of the sentence. Alternatively
```suggestion
This small bug fix release may appear insignificant, but it serves as evidence of the rock-solid quality we bring to every user and to ensure a reliable mixing experience.
```

> This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and tp ensure a reliable mixing experience.

Reviewer 2: ```suggestion
This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user to ensure a reliable mixing experience.
```

> This release improves among other things some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.

Reviewer: ```suggestion
This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
```

> You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.

Reviewer: this is not turned into a hyperlink, at least not in the preview
https://deploy-preview-303--mixxx-website.netlify.app/news/2023-08-12-mixxx-2-3-6-released/

> You can download the new version here: https://mixxx.org/download/ There you will also find the 2.4 beta with which you can already try the new features that will be released soon.

Reviewer: ```suggestion
You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
nitpick: let's keep "2.4 beta" together. &nbsp; is a hack, ^+space should be preferred AFAIU

> Here is the full change log of Mixxx 2.3.6 for more details:

Reviewer: ```suggestion
Here is the full changelog of Mixxx 2.3.6 for more details:
```

  [#11785](https://github.com/mixxxdj/mixxx/issues/11785)
  [#11789](https://github.com/mixxxdj/mixxx/issues/11789)
* Library preferences: Uncheck Serato metadata export when file metadata export is unchecked
  [#11782](https://github.com/mixxxdj/mixxx/pull/11782)
  [#11226](https://github.com/mixxxdj/mixxx/issues/11226)
* Denon MC6000MK2: Delete mapping for master gain
  [#11792](https://github.com/mixxxdj/mixxx/pull/11792)
* Improve output in case of some failed file system operations
  [#11783](https://github.com/mixxxdj/mixxx/pull/11783)
* Fix overlapping buffers when decoding M4A files using FFmpeg before 4.4
  [#11760](https://github.com/mixxxdj/mixxx/pull/11760)
  [#11545](https://github.com/mixxxdj/mixxx/issues/11545)
* Don't reject key values from file metadata with non-minor/-major scales.
  [#11001](https://github.com/mixxxdj/mixxx/pull/11001)
  [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
* Allow playing tracks with durations of more than 6 h
  [#11511](https://github.com/mixxxdj/mixxx/pull/11511)
  [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
* Soundtouch latency compensation updated for version 2.1.1 to 2.3

>   [#11154](https://github.com/mixxxdj/mixxx/pull/11154)

Reviewer: + the updated changelog from #11811 of course

This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.

> You can download the new version from the [Donload](https://mixxx.org/download/) page. There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.

Reviewer: ```suggestion
You can download the new version from the [Download](https://mixxx.org/download/) page. There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
